coalaip_bigchaindb/plugin.py

```diff
@@ -1 +1,114 @@
-# -*- coding: utf-8 -*-
+from bigchaindb_driver import BigchainDB
+from bigchaindb_driver.crypto import generate_keypair
+
+
+# FIXME: maybe this should inherit from a base plugin class provided by pycoalaip?
```

Comment:
Think makes sense at one point for sure.
What you're pointing at is basically a class that defines the interface for the ledger plugin.
As far as I know the closest there is in Python is an `AbstractBaseClass`

We shouldn't do it now though since it seems we'll likely be making some changes to this class before its finalized.